app/src/main/java/tech/ula/ServerService.kt

```diff
@@ -244,6 +246,18 @@ class ServerService : Service() {
             sessionController.findAppSession(app.name, serviceType, appsFilesystem, sessionDao)
         }
 
+        if (!username.isEmpty()) {
+            appsFilesystem.defaultUsername = username
+            appSession.username = username
+        }
+
+        if (!password.isEmpty()) {
+            appsFilesystem.defaultPassword = password
```

Comment:
It doesn't look like these changes to the filesystem are ever being propagated to the database.